controllers/usuarioController.js

```diff
@@ -0,0 +1,74 @@
+const { Usuario } = require('../models');
+
+// Obtener todos los usuarios
+exports.getAllUsuarios = async (req, res) => {
```

Comment:
En todos los casos si manejamos ingles hagamos en ingles no spaningles. uno u otro.
```suggestion
exports.getAllUsers = async (req, res) => {
```